### PR TITLE
Remove phytoref

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

Phytoref is included in PR2 5.0.0 so I removed it from DADA references option.

### CODE REVIEW

**Comment by d4straub** [comment]
Hi,
there is a recent problem with nf-core template & most recent nextflow version. I'll resolve that after https://github.com/nf-core/ampliseq/pull/702 is done. So failing tests are expected!

Also remove the entry from nextflow_schema.json please.

**Comment by danilodileo** [comment]
> Hi, there is a recent problem with nf-core template & most recent nextflow version. I'll resolve that after #702 is done. So failing tests are expected!
> 
> Also remove the entry from nextflow_schema.json please.

Hi Daniel, 
so can we just merge even if the errors are there?

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#677](https://github.com/nf-core/ampliseq/pull/677) - Added cut_its information to SDBI export
 
+- [#710](https://github.com/nf-core/ampliseq/pull/710) - Removed Phyloref from DADA2 reference option because it's part of PR2 5.0.0
```
Comment (d4straub) [inline CHANGELOG.md:20]: May be best under "removed" fine as well here.

**Review by d4straub — APPROVED** [review]
LGTM

**Comment by d4straub** [comment]
> so can we just merge even if the errors are there?

Yes, I can merge that. Just let me know and I'll do it.

**Comment by danilodileo** [comment]
> > so can we just merge even if the errors are there?
> 
> Yes, I can merge that. Just let me know and I'll do it.

It's ready to be merged!

**Pull request merged**